packages/three-mmd/src/loaders/MMDLoader.ts

```diff
@@ -457,7 +486,8 @@ class MaterialBuilder {
         : this.textureLoader
     }
 
-    const texture = loader.load(fullPath, (t) => {
+    // FIXME: wtf how does this work
+    const texture = loader.load(fullPath, (t: Texture) => {
```

Comment:
I don't understand how this worked. Maybe in real, the texture loaded callback gets called immediately?

Reply:
checked again and it is still mysterious to me..

I couldn't find a texture or loader impl that apparently supports this usage.

The closest thing I found was https://github.com/noname0310/web-mmd-viewer/blob/476504c54e44014b8096595cbe150483556a7b8e/src/asset/script/mmd/runtime/MmdParameterController.ts#L8 so I stole the name `LoadingTexture`